tests/Aspire.Qdrant.Client.Tests/AspireQdrantClientExtensionsTest.cs

```diff
@@ -0,0 +1,127 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using Aspire.Components.Common.Tests;
+using Microsoft.Extensions.Configuration;
+using Microsoft.Extensions.DependencyInjection;
+using Microsoft.Extensions.Diagnostics.HealthChecks;
+using Microsoft.Extensions.Hosting;
+using Qdrant.Client;
+using Xunit;
+
+namespace Aspire.Qdrant.Client.Tests;
+
+public class AspireQdrantClientExtensionsTest : IClassFixture<QdrantContainerFixture>
+{
+    private const string DefaultConnectionName = "qdrant";
+
+    private readonly QdrantContainerFixture _containerFixture;
+
+    public AspireQdrantClientExtensionsTest(QdrantContainerFixture containerFixture)
+    {
+        _containerFixture = containerFixture;
+    }
+
+    private string DefaultConnectionString =>
+            RequiresDockerAttribute.IsSupported ? _containerFixture.GetConnectionString() : "Endpoint=http://localhost1:6331;Key=pass";
+
+    [Theory]
+    [InlineData(true)]
+    [InlineData(false)]
+    [RequiresDocker]
+    public async Task AddQdrantClient_HealthCheckShouldBeRegisteredWhenEnabled(bool useKeyed)
+    {
+        var key = DefaultConnectionName;
+
+        var builder = CreateBuilder(DefaultConnectionString);
+
+        if (useKeyed)
+        {
+            builder.AddKeyedQdrantClient(key, settings =>
+            {
+                settings.DisableHealthChecks = false;
```

Comment:
Isn't this the default?

Reply:
Yes, we should be able to delete these lines to set it to `false`.